packages/vats/src/lib-chainStorage.js

```diff
@@ -113,3 +113,46 @@ export async function makeStorageNode(chainStorage, childName) {
   return storageNode;
 }
 harden(makeStorageNode);
+
+/**
+ * For testing, creates a chainStorage root node over an in-memory map
+ * and exposes both the map and the sequence of received messages.
+ *
+ * @param {string} rootPath
+ */
```

Comment:
Would it be useful to document the return type, or is that pointless because it's already statically known?
```suggestion
/**
 * For testing, creates a chainStorage root node over an in-memory map
 * and exposes both the map and the sequence of received messages.
 *
 * @param {string} rootPath
 * @returns {{rootNode: ReturnType<typeof makeChainStorageRoot>, data: Map<string, string>, messages: StorageMessage[]}}
 */
```

Reply:
yeah, implied by the return. I don't think it provides any useful additional safety